2025/report.md

```diff
@@ -0,0 +1,275 @@
+  - Uncategorized: 2
+- [Openfire](https://ngi.nixos.org/project/Openfire)
+  - Core: 2
+- [PagedJS](https://ngi.nixos.org/project/PagedJS)
+  - Commons: 1
+- [Re-Isearch](https://ngi.nixos.org/project/Re-Isearch)
+  - Commons: 1
+  - Review: 1
+- [ReOxide](https://ngi.nixos.org/project/ReOxide)
+  - Entrust: 1
+- [Repath-Studio](https://ngi.nixos.org/project/Repath-Studio)
+  - Commons: 1
+- [SCION](https://ngi.nixos.org/project/SCION)
+  - Uncategorized: 8
+- [Seppo](https://ngi.nixos.org/project/Seppo)
+  - Uncategorized: 1
+- [Tau](https://ngi.nixos.org/project/Tau)
+  - Core: 1
+- [ThresholdOPRF](https://ngi.nixos.org/project/ThresholdOPRF)
+  - Entrust: 1
+  - Review: 3
+- [Wax](https://ngi.nixos.org/project/Wax)
+  - Core: 1
+- [Wireguard](https://ngi.nixos.org/project/Wireguard)
+  - Uncategorized: 7
+- [holo](https://ngi.nixos.org/project/holo)
+  - Core: 1
+- [jaq](https://ngi.nixos.org/project/jaq)
+  - Uncategorized: 2
+- [kbin](https://ngi.nixos.org/project/kbin)
+  - Entrust: 2
+- [nyxt](https://ngi.nixos.org/project/nyxt)
+  - Entrust: 1
+  - Review: 2
+- [oku](https://ngi.nixos.org/project/oku)
+  - Entrust: 1
+- [openXC7](https://ngi.nixos.org/project/openXC7)
+  - Entrust: 1
+- [owasp](https://ngi.nixos.org/project/owasp)
+  - Uncategorized: 2
+- [owi](https://ngi.nixos.org/project/owi)
+  - Uncategorized: 2
+- [proximity-matcher](https://ngi.nixos.org/project/proximity-matcher)
+  - Uncategorized: 0
+- [slipshow](https://ngi.nixos.org/project/slipshow)
+  - Commons: 1
+- [stalwart](https://ngi.nixos.org/project/stalwart)
+  - Core: 1
+  - Entrust: 1
+- [verso](https://ngi.nixos.org/project/verso)
+  - Core: 2
+  - Review: 2
+- [xrsh](https://ngi.nixos.org/project/xrsh)
+  - Entrust: 1
+- [y-crdt](https://ngi.nixos.org/project/y-crdt)
+  - Commons: 1
+  - Entrust: 1
+
+### Additinal metrics
+
+#### NGIpkgs repository
+
+NGI team is maintaining
+
+- 47 services and 55 programs
+- 30 projects with demo available
+- 97 NixOS tests, associated with 97 examples
+- 80 derivations, 59 of which have an explicit update script
+
+Funded by following subgrants
+
+- 10 Commons
+- 25 Core
+- 43 Entrust
+- 50 Review
+- 89 Uncategorized
+
+#### Nixpkgs repository
+
+NGI team is maintaining 131 derivations, 107 of which have an explicit update script.
+
+### Non-packaging work
+
+FIXME: do we really need links to PRs ?
+
+In addition to a packaging work, following extra work was done:
+
+- Designed and implemented a new NGI project design (project deliverables) using
+  the NixOS module system
+
+- Implemented [service](https://github.com/ngi-nix/ngipkgs/issues/614) and
+  [program](https://github.com/ngi-nix/ngipkgs/issues/1366) demos as a practical
+  way to showcase projects
+
+- [Added update script support and bulk updates](https://github.com/ngi-nix/summer-of-nix/issues/121)
+
+- Improved project and deliverable tracking with
+  [issue templates](https://github.com/ngi-nix/ngipkgs/issues/new/choose)
```

Comment:
I think this should be ordered higher (at least number 2), because it was very important to us.